File: src/jul/operators/lidx.jl
"""
# module glidx

- Julia version: 1.8.5
- Author: tamnt
- Date: 2023-04-03

# Examples

```jldoctest
julia>
```
"""
module LIDX
   export lidx

   function lidx(IM, M, N)
      i::UInt128 = floor(IM/N)
      j::UInt128 = IM - ( N*i )
      if (j != 0)
         i += 1
      else
         j = N
      end
      return i, j
   end
end

File: src/mat/lidx.m
function [ii, jj] = lidx(M, glb_idx)

    jj = floor(glb_idx / M);
    ii = glb_idx - (M * jj);

    if (ii ~= 0)
        jj = jj + 1;
    else
        ii = M;
    end

end